pkg/autoscaler/autoscaler.go

```diff
@@ -139,21 +139,21 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (desiredPodCount
 	}
 
 	maxScaleUp := spec.MaxScaleUpRate * readyPodsCount
-	desiredStablePodCount := int32(math.Min(math.Ceil(observedStableConcurrency/spec.TargetConcurrency), maxScaleUp))
-	desiredPanicPodCount := int32(math.Min(math.Ceil(observedPanicConcurrency/spec.TargetConcurrency), maxScaleUp))
+	desiredStablePodCount := int32(math.Min(math.Ceil(observedStableValue/spec.TargetValue), maxScaleUp))
+	desiredPanicPodCount := int32(math.Min(math.Ceil(observedPanicValue/spec.TargetValue), maxScaleUp))
 
-	a.reporter.ReportStableRequestConcurrency(observedStableConcurrency)
-	a.reporter.ReportPanicRequestConcurrency(observedPanicConcurrency)
-	a.reporter.ReportTargetRequestConcurrency(spec.TargetConcurrency)
+	a.reporter.ReportStableRequestConcurrency(observedStableValue)
+	a.reporter.ReportPanicRequestConcurrency(observedPanicValue)
+	a.reporter.ReportTargetRequestConcurrency(spec.TargetValue)
 
 	logger.Debugw(fmt.Sprintf("Observed average %0.3f concurrency, targeting %0.3f.",
```

Comment:
some still left

Reply:
Rename some more. There're some left. I'll change them when we know which metric is used. Currently no way to tell and concurrency is the only one supported.